container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/backend_creator/backend_creator.go

```diff
@@ -30,24 +42,123 @@ type APIContainerModeArgs struct {
 	APIContainerIP net.IP
 }
 
-// GetLocalDockerKurtosisBackend is the entrypoint method we expect users of container-engine-lib to call
+var (
+	NoAPIContainerModeArgs *APIContainerModeArgs = nil
```

Comment:
This seems unused in this file. Let's move it to the file where it's used and make it private

Reply:
It is used in two other files and I wanted to add it to this file since it is used with a function in this file.